src/metatile_behavior.c

```diff
@@ -191,7 +191,7 @@ bool8 MetatileBehavior_IsLadder(u8 metatileBehavior)
         return FALSE;
 }
 
-bool8 MetatileBehavior_IsCaveDoor(u8 metatileBehavior)
+bool8 MetatileBehavior_IsNonAnimDoor(u8 metatileBehavior)
 {
     if(metatileBehavior == MB_CAVE_DOOR)
```

Comment:
so cave door is the only non-anim door?

Reply:
this was a rename to sync with emerald. do you prefer the previous name?